crates/validation/src/common/token_pricing.rs

```diff
@@ -13,31 +13,32 @@ use futures::StreamExt;
 use tracing::warn;
 use uniswap_v4::uniswap::{pool_data_loader::PoolDataLoader, pool_manager::SyncedUniswapPools};
 
-const BLOCKS_TO_AVG_PRICE: u64 = 5;
```

Comment:
In the Spec we set it to 5 and it shouldn't be configurable. why was this changed?

Reply:
Whenever we are working with the testnet setup you end up in a situation where you don't have enough blocks to fetch the price and on init the test nodes go boom 💥 !

Reply:
ahh makes sense. Can we change it to where you pass a optional Override value (put override as the var name) in and then have it unwrap to the constant instead so it doesn't imply that its configurable